docs/components.md

```diff
@@ -0,0 +1,20 @@
+# Components
+
+We divided the components into generic components and page-specific components.
```

Comment:
1. Instead of 'divided': 'we divide' (active voice)?
2. Perhaps change the order: first mention page-specific, then generic (so that it aligns with the order underneath)?